packages/components/src/Actions/ActionButton/ActionButton.component.js

```diff
@@ -110,13 +112,15 @@ function ActionButton(props) {
 	}
 
 	const buttonContent = getContent(props);
+	const btnIsDisabled = inProgress || disabled;
 
 	let btn = (
 		<Button
+			className={classNames({ [theme['btn-disabled']]: btnIsDisabled })}
```

Comment:
buttonProps may contains a className. please try to update buttonProps with a className if exists, else add it ?
```javascript
buttonProps.className = classNames(buttonProps.className, { [theme['btn-disabled']]: btnIsDisabled });
```